docs/integrate_ultra.md

```diff
@@ -0,0 +1,40 @@
+
+![Architecture](img/image1.png)
+
+**_High-level representation of user’s data flow between systems_**
+
+Now, we will show you the steps to follow on how to integrate your application with Ultra platform. If you face any issues or queries while performing integration, please reach out to [us](https://docs.google.com/document/d/1o3j1QRr4PofuJvWaFhsAN18vnLURFMEMBKpqD09Dt2Y/edit#heading=h.dakbw8bfy9na).
+
+## Step 1: Choose your UI platform
+
+You can build your application on Ultra using either [React Native](https://facebook.github.io/react-native/) platform or simple HTML /PWA. Ultra supports both containers and we can help you in making this choice by spotlighting a comparison between these two platforms as below:
+
+| React Native | HTML |
+|--------------|------|
+| Supports JavaScript code | Supports JavaScript code |
+| Extraordinary performance | Satisfactory performance |
+| Highly optimised for mobile applications | Average optimisation |
+| Bundle is delivered from Flipkart CDN, only differential components are downloaded and is cached within the Flipkart application | Bundle and assets follow cache-control headers and might be slower to render even from cache |
+
+### Ultra’s interaction with UI platform
+
+#### Using React Native
+
+If you choose React Native, Ultra will pull in your React Native bundle from your GitHub repository and then deliver to Flipkart application over-the-air. Flipkart receives these React Native JavaScript bundles via Dynamic Update Service ([DUS](https://github.com/Flipkart/DUS)). We recommend you to visit [this link](https://github.com/Flipkart/DUS) to understand how this works.
+
+Once DUS fetches your bundle within the Flipkart application, it will load your React Native experience within a React Fragment from where you can navigate to the other pages.
+
+This approach helps you in composing mobile applications faster having a rich UI and by using only Java scripts. You need to pick your fundamental UI building blocks and merge them simultaneously using JavaScript and React to produce a real mobile application but, limit the usage of the native bridges within Flipkart area only. Since one cannot deliver the local assets such as images/videos over-the-air as JavaScript, upload these assets to your CDN and reference them within the JavaScript code.
+
+#### Using HTML
+
+If you build your application via HTML/PWA, Ultra will launch the webpage inside android’s web-view. You can pick your existing mobile website and reuse it for Ultra after making minor changes. 
+
+This approach will help you in building a good mobile application but it will not appear as realistic and responsive as the one developed using React Native. This further does not allow you to navigate away from your main domain.
```

Comment:
> further does not allow you to navigate away from your main domain
We should put it as a note, its not a limitation of HTML. Also even on React native you cant navigate away from main domain.

Reply:
Ok, let me update it as a note